File: java/two-fer/src/main/java/Twofer.java
class Twofer {
    String twofer(String name) {
        String returnName;

        if (name != null) {
            returnName = name;
        } else {
            returnName = "you";
        }

        return String.format("One for %s, one for me.", returnName);
    }
}

File: python/two-fer/two_fer.py
def two_fer(name="you"):
    """
    Creates sentence "One for [name], one for me." with parameter name inserted

    :param name: string indicating person's name, defaults to "you"
    :return: sentence with person's name included
    """
    return "One for {}, one for me.".format(name)
